libraries/unit_converter.rb

```diff
@@ -0,0 +1,40 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: unit_converter.rb
+#
+# Module contains different functions used to manipulate the units of file sizes.
+#
+module CernerSplunk
+  SIZE_SCALE = %w(KB MB GB TB).freeze
+  # Methods converts file sizes in KB, MB, GB and TB into Bytes.
+  def self.convert_to_bytes(string)
+    fail "Unparsable size input: #{string}" if string.match(/^\d+\w{2,3}$/).nil?
```

Comment:
> CernerSplunk.convert_to_bytes('1 KB')
    RuntimeError: Unparsable size input: 1 KB
    	from (irb):12:in `convert_to_bytes'
    	from (irb):87
    	from /usr/local/var/rbenv/versions/2.1.6/bin/irb:11:in `<main>'